lib/js/components/Tooltip/Tooltip.vue

```diff
@@ -54,7 +55,7 @@ export default defineComponent({
 		tooltipParams() {
 			return {
 				position: this.placement,
-				event: this.isTouchDevice() ? 'focus' : 'hover',
+				event: isTouchDevice() ? 'focus' : 'hover',
```

Comment:
We can now remove https://github.com/bethinkpl/design-system/blob/36c10f571453bc4926dc1c2074d384445ca0e811/lib/js/components/Tooltip/Tooltip.vue#L83-L91